resources/META-INF/plugin.xml

```diff
@@ -604,8 +604,7 @@ miscellaneous:
     <errorHandler implementation="io.flutter.FlutterErrorReportSubmitter"/>
 
     <toolWindow id="Flutter Outline" anchor="right" icon="FlutterIcons.Flutter_13"
-                factoryClass="io.flutter.preview.PreviewViewFactory"
-                conditionClass="io.flutter.preview.PreviewViewCondition"/>
```

Comment:
We used to check that the project is a Flutter project.
Did we stop doing this check?

Reply:
Yes, we dropped it to eliminate cases where it should show up but didn't. IntelliJ only checks once at project open. People who have the plugin installed, but aren't doing flutter development at that moment, can just ignore this view.